napalm_ios/ios.py

```diff
@@ -369,15 +371,31 @@ def commit_config(self):
             cfg_file = self._gen_full_path(filename)
             if not self._check_file_exists(cfg_file):
                 raise MergeConfigException("Merge source config file does not exist")
-            cmd = 'copy {} running-config'.format(cfg_file)
             self._disable_confirm()
+            if confirmed:
+                # Replace running config with the rollback config
+                # And schedule the revert time
+                # If not confirmed, it will go back to the current running-config state
+                cmd = 'configure replace bootflash:{} force time {}'.format(self.rollback_cfg,
```

Comment:
Don't hard-code file system. Use `_gen_full_path(self.rollback_cfg)` to generate full path using file-system bound to object.